#! /usr/bin/env python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       name    type     level     description     default      min      max
gen.add("nms_thresh", double_t, 0, "Threshold for non-max suppression", 0.3, 0.0, 1.0)
gen.add("conf_thresh", double_t, 0, "Threshold for confidence", 0.8, 0.0, 1.0)

exit(gen.generate(PACKAGE, "jsk_perception", "FastRCNN"))
